lib/facil/meson.build

```diff
@@ -0,0 +1,71 @@
+###################################################################################
```

Comment:
Please see the [contribution guide and authorship/copyright attribution](https://github.com/boazsegev/facil.io/blob/master/CONTRIBUTING.md#license).

Copyrights and attribution should be made under my name, though it may list you as "Contributed by ..." or "Written by ...". The license shouldn't be pasted in the file (although [the file may contain a short header](https://github.com/boazsegev/facil.io/blob/fe7574d70c0bd3aa73a8d592cb902cfd6b8b8837/lib/facil/fio.h#L1-L6)).

If you don't agree to the contribution terms, let me know. Basically, I just want to avoid the mess OpenSSL had to go through when updating their license and I want to maintain control over the repo.